### PR TITLE
Fix for probate team

### PR DESCRIPTION
**Pull request by adr1ancho**
### JIRA link (if applicable) ###

configuration updates for probate team

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -5,6 +5,16 @@ provider "vault" {
 locals {
   app_full_name = "${var.product}-${var.component}"
   env_ase_url = "${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+
+  default_document_management_url = "https://api-gateway.${local.env_ase_url}.dm.reform.hmcts.net"
+  default_external_host_name = "gateway-ccd.${local.env_ase_url}.platform.hmcts.net"
```
Comment (Daniel-Lam) [inline infrastructure/main.tf]: Did you intend to omit the `https://` part?

Comment (vlaurin) [inline infrastructure/main.tf]: This needs to be the `.internal` hostname, DNS doesn't exist for this

File: infrastructure/main.tf [hunk 2]
```diff
@@ -5,6 +5,16 @@ provider "vault" {
...
+  default_cors_origin = "https://www-ccd.${local.env_ase_url}.platform.hmcts.net"
```
Comment (vlaurin) [inline infrastructure/main.tf]: This needs to be the `.internal` hostname, DNS doesn't exist for this

File: infrastructure/main.tf [hunk 3]
```diff
@@ -5,6 +5,16 @@ provider "vault" {
...
+  default_ccd_print_service_url = "https://return-case-doc-ccd.${local.env_ase_url}.platform.hmcts.net"
```
Comment (vlaurin) [inline infrastructure/main.tf]: This needs to be the `.internal` hostname, DNS doesn't exist for this

File: infrastructure/variables.tf [hunk 4]
```diff
@@ -41,13 +42,16 @@ variable "s2s_url" {
...
 
 variable "cors_origin" {
-  default = "https://www-ccd.nonprod.platform.hmcts.net"
+  type = "string"
+  default = ""
 }
 
 variable "document_management_url" {
-  default = "https://api-gateway.test.dm.reform.hmcts.net"
+  type = "string"
```
Comment (vlaurin) [inline infrastructure/variables.tf]: Please revert the above, as we are still using tactical by default for DM.

File: infrastructure/main.tf [hunk 5]
```diff
@@ -5,6 +5,12 @@ provider "vault" {
 locals {
   app_full_name = "${var.product}-${var.component}"
   env_ase_url = "${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+
+  default_app_env_ase_url = "https://${local.app_full_name}-${local.env_ase_url}"
+
+  ccd_print_service_url = "${var.ccd_print_service_url != "" ? var.ccd_print_service_url : local.default_app_env_ase_url}"
```
Comment (vlaurin) [inline infrastructure/main.tf]: This needs its own default specific to `ccd-case-print-service`

File: infrastructure/main.tf [hunk 6]
```diff
@@ -5,6 +5,12 @@ provider "vault" {
...
+  cors_origin = "${var.cors_origin != "" ? var.cors_origin : local.default_app_env_ase_url}"
```
Comment (vlaurin) [inline infrastructure/main.tf]: This needs its own default specific to `ccd-case-management-web`

File: infrastructure/main.tf [hunk 7]
```diff
@@ -5,6 +5,15 @@ provider "vault" {
 locals {
   app_full_name = "${var.product}-${var.component}"
   env_ase_url = "${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+
+  default_app_env_ase_url = "https://${local.app_full_name}-${local.env_ase_url}"
```
Comment (vlaurin) [inline infrastructure/main.tf]: I believe the `https://` prefix should be removed, as it was ```default = "gateway-ccd.nonprod.platform.hmcts.net"```

**Pull request merged**